src/main/resources/basic-file-retrievers.xml

```diff
@@ -166,7 +166,7 @@
 			<bean class="org.reactome.addlinks.dataretrieval.COSMICFileRetriever" id="COSMIC">
 				<property name="userName" value="${cosmic.user}"/>
 				<property name="password" value="${cosmic.password}"/>
-				<property name="dataURL" value="https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic/v87/CosmicMutantExport.tsv.gz" />
+				<property name="dataURL" value="https://cancer.sanger.ac.uk/cosmic/file_download/GRCh38/cosmic/v92/CosmicMutantExport.tsv.gz" />
```

Comment:
This is a similar problem as FlyBase: regular file name changes that aren't mapped to something generic like 'current'. For FlyBase, my idea for Jenkins integration was for a user input question asking for the date string in the URL and then using a grep substitution with the value in the config file (this idea can be seen in use at the new orthoinference JF: [here](https://github.com/reactome/release-orthoinference/blob/feature/74_changes/Jenkinsfile#L58), which preprends the species ID on the log files). 

Something similar could be done for COSMIC in the future. Maybe we need to start listing variable resource names.

Reply:
It would also need to work _outside_ of a Jenkins context.

Reply:
Well, at the moment we just manually update FlyBase URLs, so I was more of thinking that automatic update functionality would just be bonus functionality for Jenkins